OSImage/README.md

```diff
@@ -43,6 +43,9 @@ Available Kiwi profiles:
   * `Default-kvm`
   * `Default-dasd`
   * `Default-fba`
```

Comment:
Can you remove these profiles? All of them are for s390 and I am not sure we support s390 build hosts and thus image building.

Reply:
I want to keep the profile the same as in Build Service.

Reply:
ok, done